home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/service/impl/ApartmentInvitationServiceImpl.java

```diff
@@ -139,5 +142,17 @@ public void deactivateInvitationById(Long apartmentId, Long id) {
                         .of(pageNumber - 1, pageSize))
                 .map(invitation -> mapper.convert(invitation, ApartmentInvitationDto.class));
     }
-}
 
+    public List<ApartmentInvitation> deactivateApartmentInvitations() {
+        List<ApartmentInvitation> deactivatedApartmentInvitations = new ArrayList<>();
+
+        for (ApartmentInvitation apartmentInvitation : apartmentInvitationRepository
+                .findAll((Specification<ApartmentInvitation>) (root, criteriaQuery, criteriaBuilder) ->
+                        getInvitationForDeactivating(root, criteriaBuilder))) {
+            apartmentInvitation.setStatus(InvitationStatus.DEACTIVATED);
```

Comment:
What other statuses do we have? I think `overdue` will be more applicable here.

Reply:
done